modules/wrapping/dev.galasa.wrapping.io.grpc.java/pom.xml

```diff
@@ -19,94 +19,63 @@
 		<dependency>
 			<groupId>io.grpc</groupId>
 			<artifactId>grpc-api</artifactId>
-			<version>1.68.0</version>
+			<version>1.76.0</version>
 		</dependency>
 		<dependency>
 			<groupId>io.grpc</groupId>
 			<artifactId>grpc-context</artifactId>
-			<version>1.68.0</version>
+			<version>1.76.0</version>
 		</dependency>
 		<dependency>
 			<groupId>io.grpc</groupId>
 			<artifactId>grpc-core</artifactId>
-			<version>1.68.0</version>
+			<version>1.76.0</version>
 		</dependency>
 		<dependency>
 			<groupId>io.grpc</groupId>
 			<artifactId>grpc-netty-shaded</artifactId>
-			<version>1.68.0</version>
+			<version>1.76.0</version>
 		</dependency>
 		<dependency>
 			<groupId>io.grpc</groupId>
 			<artifactId>grpc-protobuf</artifactId>
-			<version>1.68.0</version>
-
-			<!-- 
-			This dependency has a transient dependency of this:
-			com.google.protobuf:protobuf-java:jar:3.25.3:compile
-
-			Which has a vulnerability. We will add an explicit dependency on 
-			a later version.
-			-->
-			<exclusions>
-				<exclusion>
-				<groupId>com.google.protobuf</groupId>
-				<artifactId>protobuf-java</artifactId>
-				</exclusion>
-			</exclusions>
+			<version>1.76.0</version>
 		</dependency>
 
 		<!--
 		This dependency is relied upon by io.grpc:grpc-protobuf
-		but we want to upgrade a transient dependency there.
+		but we need to depend on it directly to avoid OSGi issues.
 		-->
 		<dependency>
 			<groupId>com.google.protobuf</groupId>
 			<artifactId>protobuf-java</artifactId>
-			<version>3.25.5</version>
+			<version>3.25.8</version>
```

Comment:
There is still no version for this in the platform hence why its explicit but I'll create a new story to align all versions of this in the project and then update this.